editor.planx.uk/src/@planx/components/Checklist/Editor/Options.tsx

```diff
@@ -146,6 +167,46 @@ export const Options: React.FC<{ formik: FormikHookReturn }> = ({ formik }) => {
           editorExtraProps={{ showValueField: !!formik.values.fn }}
         />
       )}
+      {exclusiveOrOptionManagerShouldRender ? (
+        <Box mt={1}>
+          <ListManager
```

Comment:
Is there an existing prop to disable the drag handle? Maybe a nice small visual tweak we can make. Currently this single option can be dragged but not dropped anywhere.